connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/controller/InboundConnectorRestController.java

```diff
@@ -47,11 +45,17 @@ public InboundConnectorRestController(InboundExecutableRegistry executableRegist
   }
 
   @GetMapping("/inbound")
-  public List<ActiveInboundConnectorResponse> getActiveInboundConnectors(
+  public List<?> getActiveInboundConnectors(
       @RequestParam(required = false, value = "bpmnProcessId") String bpmnProcessId,
       @RequestParam(required = false, value = "elementId") String elementId,
-      @RequestParam(required = false, value = "type") String type) {
-    return getActiveInboundConnectors(bpmnProcessId, elementId, type, null);
+      @RequestParam(required = false, value = "type") String type,
+      @RequestParam(value = "groupedByConnectorId", defaultValue = "false")
+          boolean groupedByConnectorId) {
+    if (groupedByConnectorId) {
+      return getConnectorsInstances(bpmnProcessId, elementId, type);
+    } else {
+      return getActiveInboundConnectors(bpmnProcessId, elementId, type, null);
```

Comment:
Does the grouping change the type and structure of the result? If so we might want to consider a different endpoint. WDYT?